distributed-calculator/deploy/redis.yaml

```diff
@@ -5,7 +5,15 @@ metadata:
 spec:
   type: state.redis
   metadata:
```

Comment:
"...,and the Redis password with your own secret's name"

Maybe also add a link to the article that explains all this here: https://docs.dapr.io/operations/components/component-secrets/

Reply:
Updated. Thanks!